### DISCUSSION TITLE
Find a 5-digit number

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#634] [Galaxy Trivia] [answered]
Whose digits add up to 18 and whose product is 210 and the digits increase from left to right in the number.

*answer in binary*

**Comment by peterbaccaro** [comment]
12357 (binary: **11000001000101**)

18 (1 + 2 + 3 + 5 + 7 = 18)
210 (1 x 2 x 3 x 5 x 7 = 210)

**Comment by DevElkami** [comment, marked as answer]
12357 ... in binary 0011000001000101

18 (1 + 2 + 3 + 5 + 7 = 18)
210 (1 x 2 x 3 x 5 x 7 = 210)

**Comment by MohammedNayeem9** [comment]


**Reply by ParanoidUser** [reply]
Nice try! Only one small piece is missing:
   - `2 + 3 + 4 + 6 + 0` is not 18;
   - `2 * 3 * 4 * 6 * 0` is zero;
   - And the digits don't increase from left to right in the number;

But everything else is fine. 😅
